### PR TITLE
Feat/4673 add a config to disable parent space choice

### PR DESCRIPTION
**Pull request by raphj**
#4673 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

To test:
- if you already has a development.ini file please add the 467 to 472 lines from development.ini.sample to your file
- each time you change the development file, you need to re-run the backend
- please, test the configuration in the three possibilities: not set, true and false

### CODE REVIEW

**Comment by raphj** [comment]
@PhilippeAccorsi this MR introduces setting `ui.spaces.creation.parent_space_choice.enabled` (`TRACIM_UI__SPACES__CREATION__PARENT_SPACE_CHOICE__ENABLED`), which defaults to `True`. Nothing in particular needs to be done when upgrading current instances.

**Review by inkhey — APPROVED** [review]

File: frontend_app_workspace/src/container/PopupCreateWorkspace.jsx [hunk 1]
```diff
@@ -188,6 +188,8 @@ export class PopupCreateWorkspace extends React.Component {
   render () {
     const { props, state } = this
     const buttonStyleCallToAction = 'btn highlightBtn primaryColorBg primaryColorBorder primaryColorBgDarkenHover primaryColorBorderDarkenHover'
+    const isSubspacesAllowed = state.config.system.config.ui__spaces__creation__parent_space_choice__enabled
```
Comment (raphj) [inline frontend_app_workspace/src/container/PopupCreateWorkspace.jsx:191]: Suggestions:
- areSubspacesEnabled
- areParentSpacesEnabled

Comment (gbonaspetti) [reply frontend_app_workspace/src/container/PopupCreateWorkspace.jsx]: Fixed

**Review by MariePrandstetter — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
> TRACIM_UI__SPACES__CREATION__PARENT_SPACE_CHOICE__ENABLED

I not really sure if I understand corrctly what is done with this parameter.
do we have more parameter with word `ENABLED` inside name ?
`TRACIM_UI__SPACES__CREATION__PARENT_SPACE_CHOICE__ENABLED=False` => `ENABLED=False` is it clear ?

**Comment by MariePrandstetter** [comment]
tested on branch

**Pull request merged**